nessie/lakectl_util_test.go

```diff
@@ -0,0 +1,233 @@
+package nessie
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"regexp"
+	"sort"
+	"strings"
+	"testing"
+
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/require"
+)
+
+//
+// lakectl tests utility functions
+//
+var update = flag.Bool("update", false, "update golden files with results")
+
+func lakectlLocation() string {
+	return viper.GetString("lakectl_dir") + "/lakectl"
+}
+
+func lakectl() string {
```

Comment:
Actually, this is something I tried to avoid. I wanted to get as close as possible to the real lakectl format of "lakectl command arg arg", by using lakectl() + "command arg args".
When using `runCmdAndVerifyXXX(t, lakectl()+" command arg arg", <IS_TERM>,  <GOLDEN_FILE>, <VARS>)`, you can actually see the "`lakectl command arg arg`" format. This is the main reason to name lakectl() function as the tool itself
WDYT?